src/probnum/quad/solvers/bayesian_quadrature.py

```diff
@@ -49,7 +49,7 @@ def __init__(
         measure: IntegrationMeasure,
         policy: Policy,
         belief_update: BQBeliefUpdate,
-        stopping_criteria: List[StoppingCriterion],
+        stopping_criteria: List[BQStoppingCriterion],
```

Comment:
With the new `probnum.StoppingCriterion` this does not have to be a list anymore, but can just be `BQStoppingCriterion` and then the defaults below are defined via arithmetic.

Reply:
good point!

Reply:
I noted that the `bayesquad` solver used to return the type of stopping criterion that triggered in an "or" construction as info which may be quite useful for the user. With the new `LambdaStoppingCriterion` structure this is not easily possible anymore, so I removed that feature. But it might actually be quite useful in general for the user to know which of the stropping criteria the custom lambda criterion is constructed off triggered and which did not.